code/src/layouts/order-item/OrderItem.constants.ts

```diff
@@ -0,0 +1,9 @@
+import { orderStatuses } from "@/constants/orderStatuses";
+
+export const orderBadgeVariants = {
```

Comment:
Add type to make keys in this more object more restricted

Reply:
added